### ISSUE TITLE
Training Encoder On Xray Dataset

### ISSUE THREAD

**Issue by sriderya** [yuval-alaluf/restyle-encoder#44]
Hello,

First of all, thanks for your great work. I am trying to train the encoder on chest X-ray dataset. Although the results seems good, some details are missing, especially for medical sense. As can be seen from the example below, some important details such as cables are not recovered and this case is absolutely undesirable. By the way, results may seem pretty good for you but medical experts totally disagree :)

![ex-res](https://user-images.githubusercontent.com/9121907/136852814-e06ef6fe-bc31-40c4-bdfd-140e2f587dfb.jpg)

I know that you recommend moco loss for non-facial domains. In the beginning of my experiment, I used moco loss with your recommended parameters and got similar results above. Then, I used DenseNet model, trained on X-ray dataset, for extracting features while calculating id loss. I thought it would be suitable to use id loss in this conditions. I gave different lambda values to id loss, namely 0.1, 0.2 and finally 0.5. However, the results did not change significantly. The cables and artifacts was not reconstructed just like in the example above. My question here is how I can get better results? Why my results did not improve after using feature extractor trained on x-ray dataset for calculating id loss? Can you give me any feedback for my method or suggest some other methods to improve my results?

The parameters are:

{
    "batch_size": 8,
    "board_interval": 50,
    "checkpoint_path": null,
    "d_reg_every": 16,
    "dataset_type": "xray_encode",
    "delta_norm": 2,
    "delta_norm_lambda": 0.0002,
    "encoder_type": "ResNetProgressiveBackboneEncoder",
    "exp_dir": "/path/to/outdir",
    "id_lambda": 0.5,
    "image_interval": 100,
    "input_nc": 6,
    "l2_lambda": 1.0,
    "learning_rate": 0.0001,
    "lpips_lambda": 0.8,
    "max_steps": 500000,
    "moco_lambda": 0,
    "n_iters_per_batch": 5,
    "optim_name": "ranger",
    "output_size": 256,
    "progressive_start": null,
    "progressive_step_every": 2000,
    "progressive_steps": null,
    "r1": 10,
    "resume_training_from_ckpt": null,
    "save_interval": 10000,
    "save_training_data": false,
    "start_from_latent_avg": true,
    "stylegan_weights": "/path/to/stylegan2_model",
    "sub_exp_dir": null,
    "test_batch_size": 8,
    "test_workers": 8,
    "train_decoder": false,
    "update_param_list": null,
    "use_w_pool": false,
    "val_interval": 1000,
    "w_discriminator_lambda": 0,
    "w_discriminator_lr": 2e-05,
    "w_norm_lambda": 0.0,
    "w_pool_size": 50,
    "workers": 8
}

Thanks in advance...
